### PR TITLE
Fs 3731 add comment history

### PR DESCRIPTION
**Pull request by RamuniN**
https://dluhcdigital.atlassian.net/browse/FS-3731

### Change description
* Added `See History` button (available for comment owner & other assessor role users when history is available  )
* `See History` is available for all roles (`Commenter`, `Assessors` & `Lead Assessors`)

- [x] Unit tests and other appropriate tests added or updated
- [ ] README and other documentation has been updated / added (if needed)
- [x] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### How to test
_If manual testing is needed, give suggested testing steps_

### Screenshots of UI changes (if applicable)
image

image

### CODE REVIEW

File: app/blueprints/assessments/templates/macros/comments.html [hunk 1]
```diff
@@ -19,19 +19,26 @@ <h2 class="govuk-heading-m govuk-!-margin-bottom-2">Comments</h2>
...
                             application_id=comment.application_id,
                             sub_criteria_id=comment.sub_criteria_id,
+                            theme_id=comment.theme_id,
                             edit_comment="1",
                             comment_id=comment.id
                         ) }}" >Edit comment</a>
-                        {# {% if comment.updates|length > 1 %}
-                            <a class="govuk-link" href="./comment-history.html">See history</a>
-                        {% endif %} #}
+                        {% if comment.updates|length > 1 %}
```
Comment (SophieOHagan99) [inline app/blueprints/assessments/templates/macros/comments.html:33]: Does the logged in user need to be the commenter in order to view it's history

File: tests/test_authorisation.py [hunk 2]
```diff
@@ -321,6 +321,123 @@ def test_different_user_levels_see_correct_comments_on_sub_criteria_view(
...
+        response = flask_test_client.get(
+            f"/assess/application_id/{application_id}/sub_criteria_id/{sub_criteria_id}",  # noqa
+            follow_redirects=True,
+        )
+        assert response.status_code == 200
+
+        soup = BeautifulSoup(response.data, "html.parser")
+        all_comments = soup.find_all("div", class_="comment-group")
+
+        for comment in all_comments:
```
Comment (SophieOHagan99) [inline tests/test_authorisation.py:430]: Should there be an extra test to see the history of the comment (i.e. get the see history page and check it contains those two comments

**Review by SophieOHagan99 — CHANGES_REQUESTED** [review]

File: app/blueprints/assessments/templates/macros/comments.html (hunk 1 above)
Comment (RamuniN) [reply app/blueprints/assessments/templates/macros/comments.html:33]: `See History` option is available for all the roles (`Commenter`, `Assessors` & `Lead Assessors`). So logged in user must have at least one role. I think `Commenter` role is assigned by default to any assessor

File: tests/test_authorisation.py (hunk 2 above)
Comment (RamuniN) [reply tests/test_authorisation.py:430]: Good shout. Will do it

Comment (RamuniN) [reply tests/test_authorisation.py:430]: Added the tests now

**Review by SophieOHagan99 — APPROVED** [review]
LGTM

**Pull request merged**